includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -10,7 +10,13 @@
  * @since 5.4.1
  */
 class WC_Stripe_Settings_Controller {
-	public function __construct() {
+	/**
+	 * Constructor
+	 *
+	 * @param WC_Stripe_Account $account Stripe account
+	 */
+	public function __construct( WC_Stripe_Account $account ) {
+		$this->account = $account;
```

Comment:
It's probably best to also declare the property on the class' definition (also valid for the `WC_Stripe_Account`, `WC_Stripe` and `WC_Stripe_Account_Test` classes)
```suggestion
	/**
	 * The Stripe account instance.
	 *
	 * @var WC_Stripe_Account
	 */
	private $account;

	/**
	 * Constructor
	 *
	 * @param WC_Stripe_Account $account Stripe account
	 */
	public function __construct( WC_Stripe_Account $account ) {
		$this->account = $account;
```